src/Nox.Types/Common/Enumeration.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Nox.Types;
```

Comment:
Can we change the namespace to Nox.Types.Common?

Reply:
Required changes were applied